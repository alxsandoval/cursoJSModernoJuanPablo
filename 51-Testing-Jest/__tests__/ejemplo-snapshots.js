// Snapshots: Son datos que se almacenan en un string
// Para actualizar el snapshot: npm t -- -u

const cliente = {

    nombre: 'Alexis 2',
    balance: 500,
    tipo: 'Premium'

};

// Grupo de pruebas
describe('Testing al cliente', ()=>{

    // Prueba 1
    test('Es Alexis', ()=>{

        expect( cliente ).toMatchSnapshot();

    });

});
