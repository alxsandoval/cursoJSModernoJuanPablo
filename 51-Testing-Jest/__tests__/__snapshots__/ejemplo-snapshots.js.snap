// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Alexis 1`] = `
{
  "balance": 500,
  "nombre": "Alexis 2",
  "tipo": "Premium",
}
`;
